### PR TITLE
feat: jira plugin page

### PR DESCRIPTION
**Pull request by sjaanus**
image

image

### CODE REVIEW

File: frontend/src/component/menu/routes.ts
Comment (sjaanus) [inline frontend/src/component/menu/routes.ts:342]: Currently this serves only static page on Jira, but if more pages come, there might be possibility to make it generic.

File: frontend/src/component/common/FormTemplate/FormTemplate.tsx
Comment (thomasheartman) [inline frontend/src/component/common/FormTemplate/FormTemplate.tsx:204]: This will surely work just fine for now, but is there a world wherein the `apiDisabled` variable gets somehow uncoupled from `formatApiCode`, so that the function tries to force `formatApiCode` when it's undefined? Would it be better to just check for that value directly within the component?

Something like:
```tsx
if (!formatApiCode) return null
else return fullcomponent
```

Absolutely not a blocker, though! Just a thought I had.

File: frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx
Comment (thomasheartman) [inline frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx:39]: What HTML elements is this rendered as (and what kind of a thing is it)? From the context, it sounds as if it might be an image with a caption? In that case, should we use the `figure` and `caption` elements? And should the `title` be an `h3` or something similar (depending on its position in the page)? If you wanna be very nitty gritty, the StyledContainer could probably be an `article` instead of a `div`, too, but that's not very important.

Also, the images don't have any captions or alt text, which is an accessibility issue. Based on the usage in the new code, I think the typography should be put in a `figcaption` component, linked to the images `figure` component.

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (thomasheartman) [inline frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:59]: Is this nesting `<p>` tags? I don't think that's valid HTML (and it's quite possible your console is screaming at you - I noticed mine was shouting at me yesterday). 

Putting block-level elements, such as `ul` within `p` tags is also not valid HTML, according to [MDN's article on `p`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p)

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (thomasheartman) [inline frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:93]: Yeah, based on my understanding of the image container now, I think this would be better served as a [figure with a figcaption element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure). That fig caption takes the place of the alt text in this case too, so you don't need to think about that.

**Review by thomasheartman — APPROVED** [review]
No blockers, happy for you to merge this if it gives you more momentum 😄 However, I do have a few points that I think should be addressed either before merging or in a follow-up. Musings on whether variables will be present are completely optional and just thoughts I have. But the points on valid HTML structure and accessibility I think should be taken into account and fixed. 

Happy to take any questions or hear thoughts ☺️

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (sjaanus) [reply frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:93]: @thomasheartman changed to figure. I am not sure if title and description both should belong to figcaption, but based on googleing I think yes.

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (sjaanus) [reply frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:59]: Fixed

File: frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx
Comment (sjaanus) [reply frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx:39]: Done

Comment (thomasheartman) [reply frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx:39]: Did you push the changes? For some reason, I don't think I'm seeing them? 🤔

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (thomasheartman) [reply frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:93]: Hmm, title in the figcaption? Yeah, it's permitted content, so I think that makes sense 😄

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (thomasheartman) [reply frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:59]: Same with this: I'm not seeing the changes. But maybe github is slow? Or maybe you haven't pushed yet? 😅

File: frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx
Comment (thomasheartman) [inline frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx:17]: Based on the usage (and my understanding from the screenies you posted), it feels like this should be an `h2` element if I'm reading it right? But that's a minor thing, and I'll leave it up to you, whether or not to handle that.

```suggestion
export const StyledTitle = styled('h2')({
```

File: frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx
Comment (thomasheartman) [reply frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx:59]: Yeah, there they are!

**Review by thomasheartman — APPROVED** [review]
Super; much better 😄 🙌🏼

**Pull request merged**
